frontend/public/components/data-view-poc/DataViewPodList.tsx

```diff
@@ -0,0 +1,801 @@
+        id: podColumnInfo.status.id,
+        props: { className: podColumnInfo.status.classes },
+        cell: <PodStatus pod={pod} />,
+      },
+      [podColumnInfo.ready.id]: {
+        id: podColumnInfo.ready.id,
+        props: { className: podColumnInfo.ready.classes },
+        cell: `${readyCount}/${totalContainers}`,
+      },
+      [podColumnInfo.restarts.id]: {
+        id: podColumnInfo.restarts.id,
+        props: { className: podColumnInfo.restarts.classes },
+        cell: restarts,
+      },
+      [podColumnInfo.owner.id]: {
+        id: podColumnInfo.owner.id,
+        props: { className: podColumnInfo.owner.classes },
+        cell: showNodes ? (
+          <ResourceLink kind="Node" name={pod.spec.nodeName} namespace={namespace} />
+        ) : (
+          <OwnerReferences resource={pod} />
+        ),
+      },
+      [podColumnInfo.memory.id]: {
+        id: podColumnInfo.memory.id,
+        props: { className: podColumnInfo.memory.classes },
+        cell: bytes ? `${formatBytesAsMiB(bytes)} MiB` : '-',
+      },
+      [podColumnInfo.cpu.id]: {
+        id: podColumnInfo.cpu.id,
+        props: { className: podColumnInfo.cpu.classes },
+        cell: cores ? t('public~{{numCores}} cores', { numCores: formatCores(cores) }) : '-',
+      },
+      [podColumnInfo.created.id]: {
+        id: podColumnInfo.created.id,
+        props: { className: podColumnInfo.created.classes },
+        cell: <Timestamp timestamp={creationTimestamp} />,
+      },
+      [podColumnInfo.node.id]: {
+        id: podColumnInfo.node.id,
+        props: { className: podColumnInfo.node.classes },
+        cell: <ResourceLink kind="Node" name={pod.spec.nodeName} namespace={namespace} />,
+      },
+      [podColumnInfo.labels.id]: {
+        id: podColumnInfo.labels.id,
+        props: { className: podColumnInfo.labels.classes },
+        cell: <LabelList kind={kind} labels={labels} />,
+      },
+      [podColumnInfo.ipaddress.id]: {
+        id: podColumnInfo.ipaddress.id,
+        props: { className: podColumnInfo.ipaddress.classes },
+        cell: pod?.status?.podIP ?? '-',
+      },
+      [podColumnInfo.traffic.id]: {
+        id: podColumnInfo.traffic.id,
+        props: { className: podColumnInfo.traffic.classes },
+        cell: <PodTraffic podName={name} namespace={namespace} />,
+      },
+    };
+    const dataViewRow: DataViewTd[] = columns.map(({ id }) => rowCells[id]);
+    const actionsRow: DataViewTd = {
+      id: '',
+      props: { className: Kebab.columnClass },
+      cell: <LazyActionMenu context={context} isDisabled={phase === 'Terminating'} />,
+    };
+    // Always add the actions column
+    dataViewRow.push(actionsRow);
+    return dataViewRow;
+  });
+}
+
+/**
+ * The sorting logic was copied over from public/components/factory/Table/VirtualizedTable.tsx file
+ * For the most part it is 1:1
+ * column removal based on screen size was omitted from POC
+ * the columnShift attribute was omitted from the POC
```

Comment:
It appears [`columnShift` was added for the local storage operator plugin](https://github.com/openshift/console/pull/5014/files#diff-88910eaad88244e5a1a05c0af91ff4e2691f9b916abf4a77575db876414b1b15), which has since moved out of the repo, and I don't see any other instances that are using it, so I think it is ok to omit for now.